python/IECoreMaya/Menu.py

```diff
@@ -106,7 +106,7 @@ def _parseDefinition( self, parent, definition ):
 		if not isinstance( definition, IECore.MenuDefinition ):
 			raise IECore.Exception( "Definition is not a valid IECore.MenuDefinition object." )
 
-		allPaths = dict(definition.items()).keys()
+		allPaths = list(dict(list(definition.items())).keys())
```

Comment:
lol

Reply:
addressed in 06d9eabf05b10b2ac9bfbc78a076ca7717d5e400